export-image/prerun.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
-
+set -x
```

Comment:
This is used to print commands and their arguments as they are executed.
If you want to start debugging, you can ssh into the docker container and set it there.

Reply:
Usually option PRESERVE_CONTAINER=1 is not used from space considerations. If this stage fails, those printings give us extra information. 
Otherwise, we'll need to re-build the image from scratch, waiting few hours to build until this stage. And since the issue appears randomly, we may not catch it.

Reply:
PRESERVE_CONTAINER Is used in the build-docker.sh, after deploying.
It won't matter if the variable is set or not because if the build fails at a certain step, the script won't continue (and the container will not get erased).